client/components/_tests_/header.test.tsx

```diff
@@ -0,0 +1,25 @@
+import Header from '../header';
+import { render } from '@testing-library/react';
+
+describe("header test", () => {
+  render(<Header />);
+  const links = Array.from(document.querySelectorAll('#link'))
+  const linksContent = links.map(a => a.innerHTML)
+
+  it("is rendering header's tag", () => {
+    const header = document.querySelector("#header")
+    expect(header.getAttribute('id')).toBe('header')
+  })
+
+  it.each(links)("is checking header's links path exists", (a) => {
```

Comment:
Rename a to link

Reply:
i named it as `links` 'cause i get all the links at once, you thinking is more appropriated name it as `link` even if it has more then one link?

Reply:
done